third_party/bazel/cuda_static/cuda_static_configure.bzl

```diff
@@ -0,0 +1,20 @@
+load("//bazel:common.bzl", "files_exist", "get_bash_bin")
+
+_TF_CUDA_HOME = "TF_CUDA_HOME"
```

Comment:
A little bit worry about inconsistence on env var prefix `TF_`. This file is expect to be a common tool, if torch_blade reference it and has to write a `TF_`-prefixed env var, which is counterintuitive.

Reply:
As I know, torch blade also uses the bazel rules for gpus/cuda etc from tf community, I think this may not be a problem now ...